Adds descriptions to all cargo orders!

MrDoomBringer opened the pull request:
i sacrifice sweet sweet gbp to bring you, the players, nice fluffy text

:cl: MrDoomBringer
add: Orderable supplies in cargo now all have descriptions! The station's overall FLAVORFUL_TEXT stat has gone up by nearly 2% as a result.
/:cl:

[why]: so people know what they waste their credits on.
also deletes some duplicate orders from #35970

MrDoomBringer commented:
fuck im retarded gimme a bit to rebuild tgui and erase my mistakes

81Denton commented:
How many GBP do I have to pay you to do the same for the mining vendor?

MrDoomBringer commented:
@81Denton 
![image](https://user-images.githubusercontent.com/29008542/36903837-dad92fea-1dfc-11e8-83fe-b7734c6aeb00.png)

81Denton commented:
One MILLION GBP?

MrDoomBringer commented:
yes

F-OS commented on code/modules/cargo/packs.dm:
```diff
@@ -1695,6 +1744,7 @@
 
 /datum/supply_pack/misc/foamforce/bonus
 	name = "Foam Force Pistols Crate"
+	desc = "psst..hey bud... remember those old foam force pistols that got discontinued for being too cool? well i got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
```
Capitalize well, anything after a !, ?, or . Is a new sentence, so capitalize that shit.

81Denton commented:
Holy shit you actually wrote individual descriptions for every single pack

Someone give this man a medal!

81Denton commented on code/modules/cargo/packs.dm:
```diff
@@ -609,6 +663,7 @@
 	
 /datum/supply_pack/engineering/engine
 	name = "Emitter Crate"
+	desc = "Useful for powering forcefield generators while destroying locked crates and intruders alike. Contains two high-powered energy emitters."
```
This one's missing the "requires CE access" line.
Also holy shit, props for writing all these descriptions.

MrDoomBringer committed:
- Introduces a non-linear component to liver damage from alcohol so weak alcohol doesn't kill you instantly. (#36089)

MrDoomBringer committed:
Merge remote-tracking branch 'upstream/master'

MrDoomBringer committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

MrDoomBringer committed:
kabbom squahsed

first half

.ract file fixes and grammer

expressconsole compatibility

uh

test

aha

gettin there

i trimmed the trailing whitespace like a mofo

pretend i didnt delete tgui

grammar

MrDoomBringer committed:
tgooey

MrDoomBringer committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

AnturK commented:
Diff looks broken.

MrDoomBringer commented:
ahhhhahhahahah fuck

MrDoomBringer committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

MrDoomBringer committed:
what the fuck is going on

MrDoomBringer committed:
tgui 2 erlectric boogkalioo

MrDoomBringer committed:
TRAVIIIIIIIIIIIIIS

duncathan commented on code/modules/cargo/console.dm:
```diff
@@ -77,7 +77,8 @@
 			"cost" = P.cost,
-			"id" = pack
+			"id" = pack,
+			"desc" = P.desc ? P.desc : P.name // If there is a description, use it. Otherwise use the pack's name.
```
replace this with `"desc" = P.desc || P.name` for a more concise version of the same behavior

duncathan commented on code/modules/cargo/expressconsole.dm:
```diff
@@ -58,7 +58,8 @@
 			"cost" = P.cost,
-			"id" = pack
+			"id" = pack,
+			"desc" = P.desc ? P.desc : P.name // If there is a description, use it. Otherwise use the pack's name.
```
same here

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -67,6 +70,7 @@
 
 /datum/supply_pack/emergency/equipment
 	name = "Emergency Bot/Internals Crate"
+	desc = "Explosions got you down? These supplies are guarenteed to patch up holes, in stations and people alike! Comes with two floorbots, two medbots, five oxygen masks and five small oxygen tanks."
```
guaranteed*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -212,6 +225,7 @@
 
 /datum/supply_pack/emergency/weedcontrol
 	name = "Weed Control Crate"
+	desc = "Keep those invasive species OUT. Contains a scythe, gasmask, and two anti-weed chemical grenades. Warrenty void if used on cannabis. Requires Hydroponics access to open."
```
replace cannabis with ambrosia imo

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -359,6 +386,7 @@
 
 /datum/supply_pack/security/armory/bulletarmor
 	name = "Bulletproof Armor Crate"
+	desc = "Contains three sets of bulletproof armor. Guarenteed to reduce a bullet's stoping power by over half. Requires Armory access to open."
```
stopping*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -367,6 +395,7 @@
 
 /datum/supply_pack/security/armory/combatknives
 	name = "Combat Knives Crate"
+	desc = "Contains three sharpened combat knives. Each knife guarenteed to fit snugly inside any Nanotrasen-standard boot. Requires Armory access to open."
```
guaranteed*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -375,6 +404,7 @@
 
 /datum/supply_pack/security/armory/ballistic
 	name = "Combat Shotguns Crate"
+	desc = "For when the enemy absolutely needs to be replaced with lead. Contains three Aussec-designed Combat Shotguns, and three Shotgun Bandoleirs. Requires Armory access to open."
```
Bandoliers* or Bandoleers*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -441,6 +478,7 @@
 
 /datum/supply_pack/security/armory/riotarmor
 	name = "Riot Armor Crate"
+	desc = "Contains three sets of heavy body armor. Advanced padding protects against close-ranged weaponry, making meele attacks feel only half as potent to the user. Requires Armory access to open."
```
melee*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -481,13 +522,15 @@
 
 /datum/supply_pack/security/armory/wt550ammo
 	name = "WT-550 Auto Rifle Ammo Crate"
+	desc = "Contains four 20-round magazines for the WT-550 Auto Rifle. Each magazines is designed to facilitate rapid tactical reloads. Requires Armory access to open."
```
Each magazine*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -506,13 +549,15 @@
 
 /datum/supply_pack/engineering/engiequipment
 	name = "Engineering Gear Crate"
+	desc = "Gear up with three toolbelts, high-visibility vests, welding helmets, hardhats, and two meson goggles!"
```
two pairs of meson goggles*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -532,32 +577,36 @@
 
 /datum/supply_pack/engineering/pacman
 	name = "P.A.C.M.A.N Generator Crate"
+	desc = "Engineers cant set up the engine? Not an issue for you, once you get your hands on this P.A.C.M.A.N. Generator! Takes in plasma and spits out sweet sweet energy."
```
can't*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -576,30 +626,34 @@
 
 /datum/supply_pack/engineering/tools
 	name = "Toolbox Crate"
+	desc = "Any robust spaceman is never far from their trusty toolbox. Contains three electircal toolboxes and three mechanical toolboxes."
```
electrical*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -625,13 +680,15 @@
 
 /datum/supply_pack/engineering/grounding_rods
 	name = "Grounding Rod Crate"
+	desc = "Four grounding rods guarenteed to keep any uppity tesla's lightning under control."
```
guaranteed*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -707,12 +769,14 @@
 
 /datum/supply_pack/engineering/bsa
 	name = "Bluespace Artillery Parts"
+	desc = "The pride of Nanotrasen Naval Command. The legendary Bluespace Artillery Cannon is a devastating feat of human engineering and testimate to wartime determination. Highly advanced research is required for proper construction. "
```
testament*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -777,54 +845,63 @@
 
 /datum/supply_pack/materials/bz
 	name = "BZ Canister Crate"
+	desc = "Contains a canister of Benzene. Requires Toxins access to open."
```
It's actually just called BZ everywhere else, do that here too

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -855,34 +935,39 @@
 
 /datum/supply_pack/materials/watertank
 	name = "Water Tank Crate"
+	desc = "Contains a tank of dihydrogen monoxide...sounds dangerous."
```
add a space after the ellipse

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1042,6 +1139,7 @@
 
 /datum/supply_pack/science/robotics/mecha_ripley
 	name = "Circuit Crate (Ripley APLU)"
+	desc = "Rip apart rocks and aliens alike with the Ripley APLU. Contains the Main Ripley control board, as well as the Ripley Peripherals board. Requires Robotics access to open."
```
replace aliens with xenomorphs

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1052,6 +1150,7 @@
 
 /datum/supply_pack/science/plasma
 	name = "Plasma Assembly Crate"
+	desc = "Everything you need to burn something to the ground, this contains three plasma assembly sets. Each set contains a plasma tank, igniter, proximity sensor, and timer! Warrenty void if exposed to high temperatures. Requires Toxins access to open."
```
Warranty*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1085,6 +1185,7 @@
 
 /datum/supply_pack/science/shieldwalls
 	name = "Shield Generator Crate"
+	desc = "These high powered Shield Wall Generators are guarenteed to keep any unwanted lifeforms on the outside, where they belong! Contains four shield wall generators. Requires Teleporter access to open."
```
guaranteed*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1122,21 +1225,9 @@
-
 /datum/supply_pack/organic/hydroponics/beekeeping_suits
 	name = "Beekeeper Suit Crate"
+	desc = "Bee buisiness booming? Better be benevolent and boost botany by bestowing bi-Beekeeper-suits! Contains two beekeeper suits and matching headwear."
```
business*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1162,141 +1254,33 @@
+
 /datum/supply_pack/organic/cream_piee
 	name = "High-yield Clown-grade Cream Pie Crate"
+	desc = "Designed by Aussec's Advanced Warfare Research Division, these high-yield, Clown-grade cream pies are powered by a synergy of performance and efficiency. Guarenteed to provide maximum results."
```
Guaranteed*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1162,141 +1254,33 @@
-
 /datum/supply_pack/organic/hydroponics
 	name = "Hydroponics Crate"
+	desc = "Supplies for growing a great garden! Contains two bottles of ammonia, two plant-b-gone spray bottles, a hatchet, cultivator, plant analyzer, as well as a pair of leather gloves and a botanist's apron."
```
Plant-B-Gone*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1375,6 +1335,7 @@
 
 /datum/supply_pack/organic/pizza
 	name = "Pizza Crate"
+	desc = "Best prices on this side of the galaxy. All deliveries are guarenteed to be 99% anomaly-free!"
```
guaranteed*

duncathan replied:
you missed this one

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1408,6 +1369,7 @@
 
 /datum/supply_pack/organic/potted_plants
 	name = "Potted Plants Crate"
+	desc = "Spruce up the station with these lovley plants! Contains a random assortment of five potted plants from Nanotrasen's potted plant research division. Warrenty void if thrown."
```
lovely*

duncathan replied:
Warranty*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1457,6 +1421,7 @@
 
 /datum/supply_pack/organic/critter/butterfly
 	name = "Butterflies Crate"
+	desc = "Not a very dangerous insect, but they do give off a better image than say, flies or cockroaches."//is that a motherfucking worm reference
```
... than, say, flies ...

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1484,12 +1450,27 @@
 
 /datum/supply_pack/organic/critter/chick
 	name = "Chicken Crate"
+	desc = "The chiken goes bwaak!"
```
ok come on man how'd you fuck this one up

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1484,12 +1450,27 @@
 
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	desc = "CRAAAAAAB ROCKET. CRAB ROCKET. CRAB ROCKET.  CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB ROCKET. CRAFT. ROCKET. BUY. CRAFT ROCKET. CRAB ROOOCKET. CRAB ROOOOCKET. CRAB CRAB CRAB CRAB CRAB CRAB CRAB CR ...
```
there's an extra space after the third period

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1504,40 +1485,46 @@
 
 /datum/supply_pack/organic/critter/goat
 	name = "Goat Crate"
+	desc = "The goat goes baa! Warrenty void if used as a replacement for Pete."
```
Warranty*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1504,40 +1485,46 @@
 
 /datum/supply_pack/organic/critter/pug
 	name = "Pug Crate"
+	desc = "Like a normal dog, but...squished. Comes with a nice collar!"
```
but... squished.

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1604,13 +1596,15 @@
 
 /datum/supply_pack/misc/bicycle
 	name = "Bicycle"
+	desc = "Nanotrasen reminds all employees to never toy with powers outside their control"
```
add a period

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1627,6 +1671,7 @@
 
 /datum/supply_pack/misc/paper
 	name = "Bureaucracy Crate"
+	desc = "High stacks of papers on your desk Are a big problem - make it Pea-sized with these bureacratic supplies! Contains six pens, some camera film, hand labeler supplies, a paper bin, three folders, two clipboards and two stamps."//that was too forced
```
Are Pea

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1658,12 +1654,14 @@
 
 /datum/supply_pack/misc/randomised/contraband
 	name = "Contraband Crate"
+	desc = "psst.. bud... want some contraband? I can get you a poster, some nice cigs, bling, even some ambrosia deus...you know, the good stuff. Just keep it away from the cops, kay?"
```
Psst...*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1702,6 +1702,7 @@
 
 /datum/supply_pack/misc/foamforce/bonus
 	name = "Foam Force Pistols Crate"
+	desc = "psst..hey bud... remember those old foam force pistols that got discontinued for being too cool? Well I got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
```
Psst... *

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1783,20 +1790,23 @@
 
 /datum/supply_pack/misc/mule
 	name = "MULEbot Crate"
+	desc = "Pink-haired Quartermaster not doing her job? Replace them with this tireless worker, today!"
```
Replace her*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1810,6 +1820,7 @@
 
 /datum/supply_pack/misc/janitor/lightbulbs
 	name = "Replacement Lights"
+	desc = "May the light of Aether shine upon this station! Or at least, the light of forty two light tubes and twenty one light bulbs."
```
is this a metroid prime 2 reference if so i love you

MrDoomBringer replied:
yeaaah buddy

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1870,6 +1885,7 @@
 
 /datum/supply_pack/misc/costume_original
 	name = "Original Costume Crate"
+	desc = "Reenact shakespearean plays with this assortment of outfits. Contains eight different costumes!"
```
Shakespearean*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1920,6 +1937,7 @@
 
 /datum/supply_pack/misc/wizard
 	name = "Wizard Costume Crate"
+	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Nanotrasen would like to remind it's employees that actually joining the Wizard Federation is subject to termination of job and life."
```
its*

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1962,6 +1981,7 @@
 
 /datum/supply_pack/misc/randomised/toys
 	name = "Toy Crate"
+	desc = "Who cares about pride and accomplishment? Skip the gaming and get straight to the sweet rewards with this product! Contains five random toys. Warrenty void if used to prank research directors."
```
Warranty*

duncathan reviewed (changes requested):
mostly just spelling or formatting mistakes. you've done a great job capturing the tone imo, well done!

MrDoomBringer committed:
DUNC I LOVE YOU

duncathan commented on code/modules/cargo/packs.dm:
```diff
@@ -1888,6 +1904,7 @@
 
 /datum/supply_pack/misc/formalwear
 	name = "Formalwear Crate"
+	desc = "You're gonna like the way you look, I guarentee it. Contains an asston of fancy clothing."
```
I missed this one

MrDoomBringer committed:
guaranteed

MrDoomBringer commented:
![image](https://user-images.githubusercontent.com/29008542/37249156-b83ec418-24af-11e8-8bc4-958e0e8c024c.png)
